modules/util/copy.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2013 com authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License"): you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// Copyright 2020 The Gitea Authors. All rights reserved.
```

Comment:
I'm not sure how to deal with the License header since both func are copied from github.com/unknwon/com ...

Reply:
I think just copy the original file's copyright head to this file should be OK.